policyengine_us/tests/policy/baseline/gov/states/ny/ny_sun_solar_test.yaml

```diff
@@ -0,0 +1,119 @@
+- name: test_1.1_NY SUN
```

Comment:
Could you make the tests a bit more descriptive, just briefly explaining what edge case we are testing for